fastlane/lib/fastlane/actions/apple_pay_cert.rb

```diff
@@ -0,0 +1,154 @@
+        return x509_cert_path
+      end
+
+      def self.certificate
+        Spaceship.certificate.apple_pay_certificate
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.is_supported?(platform)
+        [:ios].include?(platform)
+      end
+
+      def self.description
+        'Creates an Apple Pay certificate'
+      end
+
+      def self.details
+        "This action allows you to create an Apple Pay merchant certificate."
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :username,
+                                       env_name: "APPLE_PAY_USERNAME",
+                                       description: "Your Apple ID Username"),
+          FastlaneCore::ConfigItem.new(key: :merchant_bundle_id,
+                                   env_name: "APPLE_PAY_MERCHANT_BUNDLE_ID",
+                                   description: "You merchant bundle identifier (e.g: merchant.com.example)"),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                 short_option: "-b",
+                                 env_name: "APPLE_PAY_CERT_TEAM_ID",
+                                 code_gen_sensitive: true,
+                                 default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                 default_value_dynamic: true,
+                                 description: "The ID of your Developer Portal team if you're in multiple teams",
+                                 optional: true,
+                                 verify_block: proc do |value|
+                                   ENV["FASTLANE_TEAM_ID"] = value.to_s
+                                 end),
+          FastlaneCore::ConfigItem.new(key: :p12_password,
+                                     short_option: "-p",
+                                     env_name: "APPLE_PAY_CERT_P12_PASSWORD",
+                                     sensitive: true,
+                                     description: "The password that is used for your p12 file",
+                                     default_value: ""),
+          FastlaneCore::ConfigItem.new(key: :team_name,
+                                       short_option: "-l",
+                                       env_name: "APPLE_PAY_CERT_TEAM_NAME",
+                                       description: "The name of your Developer Portal team if you're in multiple teams",
+                                       optional: true,
+                                       code_gen_sensitive: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_name),
+                                       default_value_dynamic: true,
+                                       verify_block: proc do |value|
```

Comment:
same as https://github.com/fastlane/fastlane/pull/12466/files#r265621743

Reply:
`FASTLANE_TEAM_NAME` is equal to `APPLE_PAY_CERT_TEAM_NAME`